composer.json

```diff
@@ -1,44 +1,18 @@
 {
     "name": "govcms/govcms-project",
-    "description": "Project template for GovCMS Drupal 8 projects with composer",
+    "description": "GovCMS Drupal 8 Scaffold",
     "type": "project",
     "license": "GPL-2.0-or-later",
-    "repositories": [
-        {
+    "repositories": {
+        "govcms": {
             "type": "composer",
-            "url": "https://packages.drupal.org/8"
+            "url": "https://satis.govcms.gov.au"
```

Comment:
* Is Satis still a POC - are we now in a position to advertise it?
* Is this formalized as part of security assessments - not an issue as long as it's formalized

Reply:
This is a blocker for merge. Architecturally everything is in the right place, and it has been road tested (by me). But yes it should be formalised.